Makefile

```diff
@@ -136,6 +137,17 @@ $(TEST): $(SOURCE) $(SPECS) $(BIN)
 testem:
 	$(BIN)/testem
 
+# Rules to attach checksums to JavaScript source in the header
+app.html: app.html.in .make-script-block
+	sed -e '/%SCRIPTS%/ {' -e 'r .make-script-block' -e 'd' -e '}' app.html.in > app.html
+
+.make-script-block: $(SOURCE)
+	> .make-script-block
+	for file in $(SOURCE); do \
+		checksum=$$(md5sum $$file | cut -c -32) ; \
```

Comment:
D'oh! I'd tried to use awk earlier and thought it didn't work because I forgot to add the extra $ in $$1